Jenkinsfile

```diff
@@ -5,36 +5,40 @@
 
 standardProperties()
 
-tinyPods.node() {
-  stage("clean") {
-    exec('yarn clean')
-  }
+timestamps {
+  tinyPods.node() {
+    stage("clean") {
+      exec('yarn clean')
+    }
 
-  stage("install") {
-    yarnInstall()
-  }
+    stage("install") {
+      yarnInstall()
+    }
 
-  stage("build") {
-    exec('yarn build')
-  }
+    stage("build") {
+      exec('yarn build')
+    }
 
-  stage("test") {
-    exec('yarn test')
-
-    bedrockBrowsers(
-      prepareTests: {
-        yarnInstall()
-        exec('yarn build')
-      },
-      testDirs: [ 'modules/sample/src/test/ts/**/pass' ],
-      custom: '--config modules/sample/tsconfig.json --customRoutes modules/sample/routes.json --polyfills Promise Symbol'
-    )
-  }
+    stage("test") {
+      exec('yarn test')
+
+      bedrockBrowsers(
+        prepareTests: {
+          yarnInstall()
+          exec('yarn build')
+        },
+        testDirs: [ 'modules/sample/src/test/ts/**/pass' ],
+        custom: '--config modules/sample/tsconfig.json --customRoutes modules/sample/routes.json --polyfills Promise Symbol'
+      )
+    }
 
-  if (isReleaseBranch()) {
-    stage("publish") {
-      tinyNpm.withNpmPublishCredentials {
-        exec('yarn lerna publish from-package --yes --ignore @ephox/bedrock-sample')
+    if (isReleaseBranch()) {
+      stage("publish") {
+        tinyNpm.withNpmPublishCredentials {
+          // We need to tell git to ignore the changes to .npmrc when publishing
+          exec('git update-index --assume-unchanged .npmrc')
```

Comment:
Don't know enough about lerna. So in this case it willl git add all files and commit them on publish?

Reply:
It doesn't actually end up doing anything with committing or pushing files and instead only tags things for the `from-package` because it's purpose is to just use what's already in git. Other variants of publish do though and the publish command as a whole will just try to check for uncommitted changes.

Anyways, the point of this addition is to make Git believe there are no actual changes to the npmrc file.